lib/rhc/commands/app.rb

```diff
@@ -376,9 +376,20 @@ def tidy(app)
     syntax "<app> [--namespace NAME]"
     takes_application :argument => true
     def enable_ha(app)
-      app_action :enable_ha
+      rest_app = find_app
+
+      scaleup_needed = false
+      gear_groups = rest_app.gear_groups
+      gear_groups.each do |gg|
+        scaleup_needed = true if gg.attributes["scales_from"] > 1
```

Comment:
We don't want to scale a user's application up or down during this command. It will be best to let the user know that they need to scale up, or that they can scale down and back up to get the two haproxy cartridges and remain at their same scale. 

I'll modify the PR to include that extra bit of information.